lib/workers/repository/onboarding/pr/index.ts

```diff
@@ -97,12 +98,12 @@ If you need any further assistance then you can also [request help here](${confi
   // istanbul ignore if
   if (config.prHeader) {
     const prHeader = String(config.prHeader || '');
-    prBody = `${prHeader}\n\n${prBody}`;
+    prBody = template.compile(prHeader, config) + `\n\n${prBody}`;
```

Comment:
Can't you do the templating inside the template string?

Reply:
Sorry, i don't understand.
Could you do a suggestion ?
Is the same as lineno 106
```
prBody = `${prBody}\n---\n\n` + template.compile(prFooter, config);
```

Reply:
Why are you mixing concatenation and a template string (with backticks)?

Reply:
you prefer ?
```
prBody = `${template.compile(prHeader, config)}\n\n${prBody}`;
````

Reply:
```suggestion
    prBody = `${template.compile(prHeader, config)}\n\n${prBody}`;
```

Reply:
done in fix: review